operator/pkg/controllers/psstrategy/worker.go

```diff
@@ -39,11 +38,12 @@ func newWorkerManager() *WorkerManager {
 }
 
 func (m *WorkerManager) generateWorker(job *elasticv1alpha1.ElasticJob, replicaIndex int32) *corev1.Pod {
-	workerSpec := job.Spec.ReplicaSpecs[ReplicaTypeWorker]
-	workerTemplate := &workerSpec.Template
-	workerName := m.generatePodName(job, replicaIndex)
-	pod := m.GeneratePod(job, workerTemplate, workerName)
-	pod.Labels[LabelRestartCount] = fmt.Sprintf("%d", workerSpec.RestartCount)
+	spec := job.Spec.ReplicaSpecs[ReplicaTypeWorker]
+	name := m.generatePodName(job, replicaIndex)
+	pod := m.GeneratePod(job, &spec.Template, name)
+	pod.Labels[LabelRestartCount] = fmt.Sprintf("%d", spec.RestartCount)
```

Comment:
Labels can be empty

Reply:
`GeneratePod` has check it.
https://github.com/intelligent-machine-learning/easydl/blob/1ea504b4a39e52279b02885ff5e8d77499f7b9d7/operator/pkg/controllers/pod.go#L44-L46